docs/root/overview.md

```diff
@@ -189,3 +189,9 @@ other formats (e.g. human, fortio). It can be very useful to always store the
 json output format, yet be able to easily get to one of the other output
 formats. It’s like having the cake and eating it too!
 
+## User-specified Nighthawk logging
+
+Users can use their own Nighthawk logging options when calling **ENVOY_LOG**.
+Basically it would require two steps: 1) Create a logging sink delegate
+inherited from [Envoy SinkDelegate](https://github.com/envoyproxy/envoy/blob/master/source/common/common/logger.h). 2) Constructor a ServiceImpl object with
```

Comment:
Looking at [how this gets rendered](https://github.com/envoyproxy/nighthawk/blob/f8dfdd790655e6da01146b005ad7d5989d8dd34f/docs/root/overview.md), I feel specifying a markdown list here may enhance readability. E.g.

```
1. Create a logging sink delegate
2. ....
3. ....
```

Reply:
Thanks Otto! Please see this [updated version](https://github.com/envoyproxy/nighthawk/blob/701ab40f99e70c2e6443f6eafef4fc4bb0b507b9/docs/root/overview.md).